File: PL034.cpp
#include <iostream>
#include <bits/stdc++.h>
using namespace std;

vector<int> getMultiplication(int n)
{
    vector<int> v;
    for (int i = 1; i < 11; i++)
    {
        v.push_back(n * i);
    }

    return v;
}

int main()
{
    int n;
    cout << "Enter the number : " << endl;
    cin >> n;
    cout << "Below is the multiplication of the number " << n << endl;
    vector<int> result = getMultiplication(n);
    for (int i : result)
    {
        cout << i << " ";
    }

    cout << endl;
    return 0;
}

File: PL034.java
import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class PL034 {
    public static void main(String[] args) {
        int n;
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the number : ");
        n = scanner.nextInt();
        scanner.close();
        System.out.println("Below is the multiplication of the number : " + n);
        List<Integer> list = new ArrayList<>();
        list = getMultiplication(n);
        for (int i : list) {
            System.out.print(i + " ");
        }
    }

    public static List<Integer> getMultiplication(int n) {
        List<Integer> list = new ArrayList<>();

        for (int i = 1; i < 11; i++) {
            list.add(n * i);
        }

        return list;
    }
}
